doc/api/buffer.markdown

```diff
@@ -638,9 +609,9 @@ console.log(buf.length);
 ### buf.readDoubleBE(offset[, noAssert])
 ### buf.readDoubleLE(offset[, noAssert])
 
-* `offset` Number
-* `noAssert` Boolean, Optional, Default: false
-* Return: Number
+* `offset` {Number} `0 <= offset <= buf.length`
```

Comment:
Well, I'd say that anybody reading documentation on a module designed to do binary stuff "techy" :P 

It should be fairly obvious and intuitive that one cannot read (or write) out of range, so I'd argue against having a separate paragraph for such a simple constraint.

How about:

> * `offset` Number Must be between 0 and the length of the buffer, inclusive

I don't think it is any clearer than the pseudo-JavaScript expression however.